src/Microsoft.ML.CodeGenerator/Templates/Console/PredictProject.cs

```diff
@@ -75,7 +75,13 @@ public virtual string TransformText()
             this.Write(this.ToStringHelper.ToStringWithCulture(Namespace));
             this.Write(".Model\\");
             this.Write(this.ToStringHelper.ToStringWithCulture(Namespace));
-            this.Write(".Model.csproj\" />\r\n  </ItemGroup>\r\n</Project>\r\n");
+            this.Write(".Model.csproj\" />\r\n  </ItemGroup>\r\n\r\n  <ItemGroup>\r\n");
+ if (Target==CSharp.GenerateTarget.Cli) {
+            this.Write("    <ProjectCapability Include=\"MLNETCLIGenerated\" />\r\n");
+}else{
+            this.Write("    <ProjectCapability Include=\"ModelBuilderGenerated\" />\r\n");
+}
```

Comment:
Should modify the upstream `.tt` so the `.cs` is formatted properly here.

Reply:
Same w/ `ModelProject.tt`

Reply:
What you mean

Reply:
@LittleLittleCloud aren't these files generated from TT files?

Reply:
In this PR it looks like the CS files were modified directly instead of changing the TT files.

Reply:
Nevermind I see them now :)

Reply:
The newly added code is currently being formatted as:
https://github.com/dotnet/machinelearning/blob/e84006b74e059d1565b2461514d37446754a25a7/src/Microsoft.ML.CodeGenerator/Templates/Console/PredictProject.cs#L77-L86


It should be:
```c#
            this.Write(this.ToStringHelper.ToStringWithCulture(Namespace));
            this.Write(".Model.csproj\" />\r\n  </ItemGroup>\r\n\r\n  <ItemGroup>\r\n");
            if (Target==CSharp.GenerateTarget.Cli) {
                this.Write("    <ProjectCapability Include=\"MLNETCLIGenerated\" />\r\n");
            }
            else {
                this.Write("    <ProjectCapability Include=\"ModelBuilderGenerated\" />\r\n");
            }
            this.Write("  </ItemGroup>\r\n</Project>\r\n");
            return this.GenerationEnvironment.ToString();
        }
```

The generated `.cs` files are ceated from their corresponding `.tt` T4 template files. The change would occur there. I left a comment on the `.cs` as it shows the output.

Reply:
@JakeRadMSFT  it's auto-gen, I didn't modify  those cs file a bit :)